### PR TITLE
Set search with Algolia

### PR DESCRIPTION
**Pull request by pierallard**
screenshot from 2016-07-05 14 24 51

### CODE REVIEW

File: _themes/akeneo_rtd/layout.html [hunk 1]
```diff
@@ -193,6 +193,20 @@
   </script>
   {% endif %}
 
+  {# ALGOLIA #}
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.js"></script>
+  <script type="text/javascript">
+    docsearch({
+      apiKey: '90e155c70f9456125f229cc924db6000',
```
Comment (ClementGautier) [inline _themes/akeneo_rtd/layout.html]: should not be set here I think. Probably set during deployment ?

File: _themes/akeneo_rtd/layout.html [hunk 2]
```diff
@@ -193,6 +193,20 @@
...
+      indexName: 'akeneo',
```
Comment (ClementGautier) [inline _themes/akeneo_rtd/layout.html]: same ?

**Comment by ClementGautier** [comment]
GTM if comments fixed

**Comment by nidup** [comment]
GTM (take care on which version you apply this change)

**Pull request merged**